src/lib/BasicAdmin.tsx

```diff
@@ -28,12 +27,12 @@ function BasicAdmin (props: any) {
   const footSpec = { foot: frameModel.part.foot, view }
 
   return (
-    <div className='BasicAdmin'>
+    <Box className='BasicAdmin'>
```

Comment:
Is Box appropriate for top level layout?

Reply:
From what I know, Box (by default) renders as a simple div with access to app's theme and `sx`. I checked other UI kits built on top of MUI, and the common approach they use is either `Box` or `styled('div')`.

Few references: 
https://github.com/codedthemes/berry-free-react-admin-template/blob/main/create-react-app/src/layout/MainLayout/index.js
https://github.com/devias-io/material-kit-react/blob/main/src/layouts/dashboard/layout.js#L10C20-L10C33
https://github.com/minimal-ui-kit/material-kit-react/blob/089f738f2e91469f0047ad3bae81b0cdb8f8dd27/src/layouts/dashboard/index.jsx

What do you recommend in this case?